### PR TITLE
Comparison of design approaches for proxy-keys

### PR DESCRIPTION
**Pull request by johnalanwoods**
Comparison of design approaches for proxy-keys, committing for posterity.

### CODE REVIEW

**Review by JaredCorduan — COMMENTED** [review]
Looks great to me! It looks like at least one image file is missing, though, `images/addresses.pdf`.

File: docs/proxy-keys/proxy-key-sols.tex [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+\end{figure}
+
+As shown above the structure of a Shelley-era address (from which one can delegate stake), is forged from a 
+\emph{payment credential}, $\mathcal{A}_p$ and a \emph{stake credential}, $\mathcal{A}_s$. With the current 
+core Cardano protocol, the payment credential, (or more accurately the credential's private key counterpart), 
+controls the spending of funds, and the stake credential controls delegation/de-delegation and withdrawal of 
+rewards from the account they accumulates to. \\
+
+\subsubsection{High-level design}
+Cognisant of the above, the solution to deliver the Proxy-key/partial delegation feature is to deposit funds
```
Comment (JaredCorduan) [inline docs/proxy-keys/proxy-key-sols.tex:251]: I'm guessing this is the British spelling of Cognizant :)

File: docs/proxy-keys/proxy-key-sols.tex [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+  \item the identity of the \gls{trustor}.
+  \item the approval of the \gls{trustor}.
+\end{itemize}
+
+This mechanism will allow us to meet most of the requirements described in the Proxy-key product requirements document,
+however, it does have some limitations and indeed some complexities to be considered.
+
+\subsubsection{Benefits \& drawbacks}
+Next we will explore the benefits and drawbacks of the Ledger based solution described above. We will also explore
+which requirements described in the PRD would be possible to meet.
```
Comment (JaredCorduan) [inline docs/proxy-keys/proxy-key-sols.tex:206]: Maybe this is just me, but what is a PRD? product requirements document?

**Comment by johnalanwoods** [comment]
> Looks great to me! It looks like at least one image file is missing, though, `images/addresses.pdf`.

Good catch!

File: docs/proxy-keys/proxy-key-sols.tex (hunk 1 above)
Comment (johnalanwoods) [reply docs/proxy-keys/proxy-key-sols.tex:251]: Do you mean best spelling 😆

File: docs/proxy-keys/proxy-key-sols.tex (hunk 2 above)
Comment (johnalanwoods) [reply docs/proxy-keys/proxy-key-sols.tex:206]: yep exactly that.

**Comment by johnalanwoods** [comment]
@JaredCorduan updated and rebased, should be GTG

File: docs/proxy-keys/proxy-key-sols.tex (hunk 1 above)
Comment (JaredCorduan) [reply docs/proxy-keys/proxy-key-sols.tex:251]: :laughing:

**Review by JaredCorduan — APPROVED** [review]
:+1: thanks for doing this @johnalanwoods !

**Pull request merged**